crates/control/src/ball_filter.rs

```diff
@@ -280,13 +295,15 @@ impl BallFilter {
             let state_prediction = constant_velocity_prediction * state_rotation;
             let control_input_model = Matrix4x2::identity();
             let odometry_translation = last_odometry_to_current_odometry.translation.vector;
-            hypothesis.moving_state.predict(
+            KalmanFilter::predict(
+                &mut hypothesis.moving_state,
```

Comment:
If you'd rename the trait function as commented below, you can revert this explicit syntax back to the `self.xxx`